ui/app/routes/vault/cluster/clients/counts/sync.ts

```diff
@@ -4,5 +4,36 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import { DEBUG } from '@glimmer/env';
 
-export default class ClientsCountsSyncRoute extends Route {}
+import type StoreService from 'vault/services/store';
+
+interface ActivationFlagsResponse {
+  data: {
+    activated: Array<string>;
+    unactivated: Array<string>;
+  };
+}
+
+export default class ClientsCountsSyncRoute extends Route {
+  @service declare readonly store: StoreService;
+
+  async getActivatedFeatures() {
+    try {
+      const resp: ActivationFlagsResponse = await this.store
+        .adapterFor('application')
+        .ajax('/v1/sys/activation-flags', 'GET');
+      return resp.data?.activated;
+    } catch (error) {
+      if (DEBUG) console.error(error); // eslint-disable-line no-console
+      return [];
```

Comment:
i decided to return an empty array here if we encounter an error fetching activated features instead of an `AdapterError` like we do [here](https://github.com/hashicorp/vault/blob/1745d50c2d6762da3fc3f306b3bb939e6ceaafe0/ui/lib/sync/addon/routes/secrets.ts#L25-L33). it didn't seem like we'd need the `AdapterError` in this case, unless there's something i'm missing?

Reply:
notes:
- it's possible for a user to not have permission for this endpoint, but the feature might still be activated. in that situation
- we can try to see if any secrets/destinations have been set up first, and if so, then allow access to this page